include/nil/crypto3/math/algorithms/make_evaluation_domain.hpp

```diff
@@ -104,6 +104,28 @@ namespace nil {
 
                 return result_type();
             }
+
+            /*!
+            @brief
+             A convenience method for choosing a basic evaluation domain
+             Returns an evaluation domain object in which the domain S has size
+             |S| >= MinSize.
+            */
+            template<typename FieldType>
+            std::shared_ptr<evaluation_domain<FieldType>> make_basic_evaluation_domain(std::size_t m) {
+                typedef std::shared_ptr<evaluation_domain<FieldType>> result_type;
+                
+                m = detail::power_of_two(m);
+
+                if (detail::is_basic_radix2_domain<FieldType>(m)) {
```

Comment:
I don't understand why we need this function if we have `make_evaluation_domain` in line 50 https://github.com/NilFoundation/crypto3-math/blob/d6070067bd21a0a1106dcc9d915dcaa8c257dd82/include/nil/crypto3/math/algorithms/make_evaluation_domain.hpp#L57 We can just send in this function m == detail::power_of_two(m); and get basic domain

Reply:
Hmm good idea haven't thought of that